docs/data/routes/docs/client-frameworks/angular/angular-tips/en.md

```diff
@@ -61,6 +61,55 @@ If you're using the default code-generated `app-components.module.ts` that's all
 export class AppComponentsModule { }
 ```
 
+You can also load multiple components from one module. This will improve code splitting of your application and will reduce the calls which need to be made for loading application resources. 
+
+For lazy-loading with multiple components you need to register two or more components with `JssModule.forChild()` method into lazy-loading module. You need to provide component name and component type.
+
+```js
+import { NgModule } from '@angular/core';
+import { JssModule } from '@sitecore-jss/sitecore-jss-angular';
+import { FirstComponent } from './first.component';
+import { SecondComponent } from './second.component';
+
+@NgModule({
+  imports: [
+    // secret sauce
+    JssModule.forChild([
+      { name: 'FirstComponent', type: FirstComponent },
+      { name: 'SecondComponent', type: SecondComponent }
+    ])
+  ],
+  declarations: [
+    MyComponent,
```

Comment:
Wouldn't the declarations need to be `FirstComponent, SecondComponent`?

Reply:
In this branch you can find my test site https://github.com/agalbulev/jss/tree/feature/test_multi_lazy. Can you send to me your configuration because I don't have this problem with my test site?

Reply:
Can you also check that the component name 'StyleguideAngularLazyLoadingComponent' is equal to the property path for this component in 'app-components.module.ts'.

Reply:
Argh I'd pasted the name of the class into the `forChild()` registration complete with the `Component` suffix. Which the JSS component name didn't have.

Chalk another one up to Angular's totally amazing error messaging :trollface: